src/DynamicRupture/Misc.h

```diff
@@ -92,6 +92,23 @@ inline double asinh(double x) { return std::log(x + std::sqrt(x * x + 1.0)); }
  * @param dip
  */
 void computeStrikeAndDipVectors(const VrtxCoords normal, VrtxCoords strike, VrtxCoords dip);
+
+/**
+ * Defines the indices under which one can find a specific quantity
+ * U, V, W: Velocities in x, y, z direction
+ * N, T1, T2: traction in normal and fault aligned directions
+ * Note: can not be an enum class, otherwise we would need to use
+ * static_cast<size_t>(dr::misc::QuanityIndices::X), whenever we want to use
+ * one of the indices as an array subscript.
+ */
+enum QuantityIndices : size_t {
```

Comment:
Maybe 
namespace quantity_indices {
enum QuantityIndices
}
Then it's not typesafe but the indices are at least not in the global namespace. And you can do using namespace, so it's quite convenient. 

Here enum class is definitely the wrong choice, because this uses enum to define a selection of constants and not a, well, enum